cmd/client.go

```diff
@@ -122,13 +125,18 @@ user-defined plans.
 				log.Fatalf("Error creating client: %v", err)
 			}
 
+			allExamples, err := server.GetAllCompleteServiceExamples(builtin.BuiltinBrokerRegistry())
```

Comment:
Rather than calling this function here to get allExamples let's create a new function based on the work you did for `TestNewExampleHandler` that will connect to the server, poll the endpoint for the server's list of examples, deserialize them into a list then pass them to `RunExamplesForService`

That way if the list on the server is different than the list the client has then the client will use whatever the server supports.

Reply:
I created the `GetExamplesFromServer()` function in `pkg/server/examples.go`, based on the tutorial here: https://blog.alexellis.io/golang-json-api-client/.